README.md

```diff
@@ -20,12 +20,13 @@ For example a simple configuration YAML (JSON is also supported) should be as fo
 ```yaml
 steps:
 - dataFrameName: df1
-  sql: 
+  checkpoint: true
```

Comment:
@amirhalatzi can you add a comment to explain what the checkpoint is used for?

Reply:
@doronporat done